src/hdr_histogram.c

```diff
@@ -352,15 +352,15 @@ void hdr_init_preallocated(struct hdr_histogram* h, struct hdr_histogram_bucket_
     h->total_count                     = 0;
 }
 
-void* hdr_aligned_calloc(size_t alignment, size_t size) {
+void* hdr_aligned_calloc(size_t num, size_t size) {
     void* memPtr = NULL;
 #ifdef _MSC_VER
-    memPtr = _aligned_malloc(size, alignment);
+    memPtr = _aligned_malloc(num * size, alignment);
```

Comment:
Does this build? `alignment` has been removed.

Reply:
errr thanks for the spot definitely a bug haven't built it on Windows yet